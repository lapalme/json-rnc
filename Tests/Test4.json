[
 {"name":"Guy","id":"Lapalme","address":45, "postalCode":"H0H 0H0","nombreOuChaineVide":45.8},
 {"id":{"no":24},"name":"Luc","address":75,"nombreOuChaineVide":""},
 {"id":true,"address":10,"name":null}
]
