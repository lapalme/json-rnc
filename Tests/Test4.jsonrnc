## a comment to skip
start = [person]
person = {name:string,
          id:(string|{no:number}),
          address:number@(minimum=10,maximum=100,exclusiveMinimum=true),
          postalCode? : cpRE,
          nombreOuChaineVide?:(number|//)
}
cpRE = /[A-Z][0-9][A-Z] [0-9][A-Z][0-9]/
