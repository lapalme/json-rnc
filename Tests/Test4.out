1:[{u'postalCode': u'H0H 0H0', u'nombreOuChaineVide': 45.8, u'address': 45, u'name': u'Gu...ess': 10}]
[2]/name	string expected:	null
true does not match any alternative:
 -[2]/id	string expected:	true
 -[2]/id	object expected:	true
[2]/address	illegal value:	10 <= 10 excl
1 objects read: 1 invalid, 0 bad, 0 with duplicate fields
Error Statistics
              1	[2]/name:string expected:
              1	true does not match any alternative:
